google/__init__.py

```diff
@@ -78,7 +78,7 @@
 install_folder = os.path.abspath(os.path.split(__file__)[0])
 user_agents_file = os.path.join(install_folder, 'user_agents.txt')
 try:
-    with open('user_agents.txt') as fp:
+    with open('user_agents_file') as fp:
```

Comment:
Right, but I don't think user_agents_file should be in quotes. We should be using the variable name, not the string.